test/integ/check_user_login_integ_test.js

```diff
@@ -0,0 +1,134 @@
+const chai = require("chai");
+const chaiHTTP = require("chai-http");
+const { stub } = require("sinon");
+const databaseOps = require("../../lib/database-ops");
+const usernameUtil = require("../../lib/util/username");
+const server = require("../../lib/server");
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const { MongoClient } = require("mongodb");
+const { assert } = chai;
+
+chai.use(chaiHTTP);
+
+// TODO:#119 shut down mongo mem server and remove --exit hopefully
+
+function getServerAgent() {
+    return chai.request.agent(server.app);
+}
+
+describe("integ", () => {
+    describe("check user login", () => {
+        var mongod = null;
+        var db = null;
+        var agent = null;
+        var usersCollection = null;
+        const TEST_USER = "test-user";
+
+        function checkUserLogin() {
+            return new Promise((resolve, reject) => {
+                agent.get("/user")
+                .send()
+                .then((res) => {
+                    resolve(res);
+                    agent.close();
+                })
+                .catch((err) => {
+                    reject(err);
+                    agent.close();
+                });
+            });
+        }
+
+        function performUserLogin() {
+            return new Promise((resolve, reject) => {
+                agent = getServerAgent();
+                agent.post("/login")
+                    .type("form")
+                    .send({
+                        "username": "test-user",
+                        "password": "test"
+                    })
+                    .then((res) => {
+                        resolve(res);
+                    })
+                    .catch((err) => {
+                        reject(err);
+                    });
+            });
+        }
+
+        it("should pass if user is logged in", () => {
+            return performUserLogin(TEST_USER)
+                .then(async () => {
+                    return checkUserLogin();     
+                })
+                .then((res) => {
+                    assert.equal(res.statusCode, 200);
+                    assert.equal(res.body.status, "success");
+                    assert.equal(res.body.message, "User Logged In");
```

Comment:
```suggestion
                    assert.equal(res.body.message, "User logged in");
```
I think for reason above the message assertion should be removed, but if you apply the other suggestion and also decide to keep the assertion, apply this suggestion too.

Reply:
Yes, it would be better if I remove message assertion as status code would be enough and  messages tend to change. Also, other integ tests implemented in code also verify through errorId and statusCode.